src/sentry/sentry_metrics/consumers/last_seen_updater.py

```diff
@@ -120,7 +121,25 @@ def __init__(
         self.__prefilter = LastSeenUpdaterMessageFilter(metrics=self.__metrics)
 
     def __should_accept(self, message: Message[KafkaPayload]) -> bool:
-        return not self.__prefilter.should_drop(message)
+        should_accept = not self.__prefilter.should_drop(message)
+
+        try:
+            parsed_message = json.loads(message.payload.value.decode("utf-8"), use_rapid_json=True)
+            org_id = parsed_message["org_id"]
+        except rapidjson.JSONDecodeError:
+            logger.error("last_seen_updater.invalid_json", exc_info=True)
+            return False
```

Comment:
You should return should_accept here instead of defaulting to `False` in case of an exception

Reply:
Oops, I forgot to change that